timescaledb/overview/release-notes/index.md

```diff
@@ -16,8 +16,10 @@ community][timescale-slack].
 
 For our next releases, we plan to add:
 
-- Continuous aggregates with compression
-- `time_bucket_ng` suport for N months and Timezones on continuous aggregates
+- Performance improvements for:
+  - Inserts and common queries on Multi-node 
+  - Continuous Aggregates by removing re-aggregation 
+- Custom origin's support in continuous aggregates with `time_bucket_ng`
```

Comment:
Is the indentation off here?

Reply:
The other 2 above are level 2.
This one is level 1.